Podverse/PVSubscriber.swift

```diff
@@ -25,12 +31,65 @@ class PVSubscriber {
         
     }
     
-    static func checkIfSubscribed(feedUrlString: String?) -> Bool {
-        if let feedUrlString = feedUrlString, let _ = Podcast.podcastForFeedUrl(feedUrlString: feedUrlString) {
+    static func unsubscribeFromPodcast(feedUrlString: String?, podcastId: String?) {
+        
+        if let feedUrlString = feedUrlString {
+            
+            // TODO: add error handling / connectivity error
+            updatePodcastOnServer(feedUrl: feedUrlString, shouldSubscribe: false) { wasSuccessful in
+                //
+            }
+
+            PVDeleter.deletePodcast(feedUrl: feedUrlString)
+            
+        }
+        
+    }
+    
+    static func checkIfSubscribed(podcastId: String?) -> Bool {
+        if let podcastId = podcastId, let _ = Podcast.podcastForId(id: podcastId) {
             return true
         } else {
             return false
         }
     }
     
+    static func updatePodcastOnServer(feedUrl:String, shouldSubscribe:Bool, completion: @escaping (_ wasSuccessful:Bool?) -> Void) {
+        
+        let urlEnding = shouldSubscribe == true ? "subscribe" : "unsubscribe"
+        
+        if let url = URL(string: BASE_URL + "podcasts/" + urlEnding) {
+            var request = URLRequest(url: url, cachePolicy: NSURLRequest.CachePolicy.reloadIgnoringLocalAndRemoteCacheData, timeoutInterval: 60)
+            request.httpMethod = "POST"
+            
+            guard let idToken = UserDefaults.standard.string(forKey: "idToken") else {
+                return
```

Comment:
might want to add a `completion(false) `here

Reply:
added in previous PR